internal/db/migrations-tern/007_blueprint_composes.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS blueprint_composes
+(
+    blueprint_id uuid REFERENCES blueprints(id) ON DELETE CASCADE,
+    compose_id uuid REFERENCES composes(job_id) ON DELETE CASCADE
```

Comment:
Does it make sense to enforce the compose_id to be unique in this table?

Reply:
Good point, it does. Now that I am thinking about this table, I am a bit ahead as creating composes through the new experimental API is not part of this PR, but if you don’t mind I will go ahead, add unique constraint and leave it here.